src/game/scenes/arena.c

```diff
@@ -1088,8 +1088,8 @@ void arena_dynamic_tick(scene *scene, int paused) {
         for(int i = 0; i < 2; i++) {
             float hp = (float)hars[i]->health / (float)hars[i]->health_max;
             float en = (float)hars[i]->endurance / (float)hars[i]->endurance_max;
-            progressbar_set_progress(local->health_bars[i], hp * 100, gs->warp_speed ? false : true);
-            progressbar_set_progress(local->endurance_bars[i], en * 100, gs->warp_speed ? false : true);
+            progressbar_set_progress(local->health_bars[i], hp * 100, gs->warp_speed || gs->clone ? false : true);
+            progressbar_set_progress(local->endurance_bars[i], en * 100, gs->warp_speed || gs->clone ? false : true);
```

Comment:
Not sure I'm reading this correct, but is the ternary operator redundant here? Would `!(gs->warp_speed || gs->clone)` do the same?

Reply:
Done.